docker-compose.yaml

```diff
@@ -0,0 +1,18 @@
+services:
+  dokku:
+    image: dokku/dokku:0.32.4
```

Comment:
thought: Ideally, we'd need to keep the version of Dokku in sync with production.